helm/prometheus-rules/templates/alerting-rules/inhibit.prometheus-agent.rules.yml

```
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: inhibit.prometheus-agent.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: inhibit.prometheus-agent
+    rules:
+    # this inhibition fires when a cluster is not running prometheus-agent.
+    # we retrieve the list of existing cluster IDs from `kube_namespace_created`
+    # then compare it with the list of deployed observability-bundles from `app_operator_app_info`
```

Should we add this for clarification?

```suggestion
    # then compare it with the list of deployed observability-bundles from `app_operator_app_info`
    # 
    # Will only produce data (and inhibitions) on MC because it's where app_operator is running
    # but that's enough to have the inhibitions on the installation-global alertmanager
```